### PR TITLE
CLOUD-2887 updates for build reports

### PR DESCRIPTION
**Pull request by sanscontext**
cc @pkennedyr @rogeliog and @pchico83 for quick review - this is already live.

Signed-off-by: LRubin <lrubin@docker.com>

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit 05a0c5554ee7fe4e8d50262c68ac32683846bbde

https://deploy-preview-612--docker-docs.netlify.com

File: docker-cloud/builds/automated-build.md [hunk 1]
```diff
@@ -124,18 +124,39 @@ should remain secret.
...
+display a **Retry** button.
 
-Both views display the pending, in progress, successful, and failed builds
-for any tag of the repository.
+![Timeline view showing the retry build button](images/retry-build.png)
 
-You can click the **Cancel** button for pending builds and builds in progress.
-If a build fails, the cancel button is replaced by a **Retry** button.
+> **Note**: If you are viewing the build details for a repository that belongs
+to an Organization, the Cancel and Retry buttons only appear if you have `Read & Write` access to the repository.
```
Comment (pkennedyr) [inline docker-cloud/builds/automated-build.md]: "build details" -> "Build Report"

**Comment by pkennedyr** [comment]
Added a couple minor comments ... otherwise LGTM.

File: docker-cloud/builds/automated-build.md (hunk 1 above)
Comment (sanscontext) [inline docker-cloud/builds/automated-build.md]: This isn't specific to the build report, thus the lower case and using the term "details"

File: docker-cloud/builds/automated-build.md [hunk 2]
```diff
@@ -124,18 +124,39 @@ should remain secret.
...
+repository and branch (or tag), the build duration, creation time and location,
+and the user namespace the build occurred in.
+
+![screen showing a build report](images/build-report.png)
+
+## Cancel or retry a build
+
+While a build is queued or running, a **Cancel** icon appears next to its Build
+Report link on the General tab and on the Builds tab. You can also click the
+**Cancel** button from the build report page, or from the Timeline tab's logs
```
Comment (sanscontext) [inline docker-cloud/builds/automated-build.md]: Discussed offline - the "build report" is a screen, not really a separate feature, and thus it doesn't need a distractingly capitalized product name.

**Comment by sanscontext** [comment]
After some discussion, standardized on a non-capitalized treatment of "build report". Merging now.

**Pull request merged**